README.md

```diff
@@ -1,5 +1,125 @@
 # Kafka Histo Streamer
 
-Consumes event data from kafka and re-streams it as histogram data
+This is a prototype to test and demonstrate Mantid being capable of live processing event data into histogram data, streaming both via Kafka. Subscribes to a topic with event data and publishes it to another topic as histogram data.
 
+## Requirements
 
+Requires Mantid (currently has to be `nightly`).
```

Comment:
Could you put the date of this (as of ...) so in 1 years time it still makes sense. Go on to say you will need a major version > 3.13